Lesson3/Program.cs

```diff
@@ -1,12 +1,21 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 
-namespace Lesson3
+namespace Project.ConsoleApp
 {
     class Program
     {
-        static void Main(string[] args)
+        [STAThread]
+        static void Main(string[] clArguments)
         {
-            Console.WriteLine("New lesson 3 command line.");
-        }
+            if (clArguments.HaveValidArguments())
+            {
+                var fileData = clArguments.GetJsonData().Concat(clArguments.GetXmlData()).ToArray();
```

Comment:
`Complexity`:  You do a `ToArray()` only to add another operation for `IEnumerable<string>` and then turn it into a list?  Why not chain them all together?